<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialog</class>
 <widget class="QDialog" name="QDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Stock Price Predictor - V1.0</string>
  </property>
  <widget class="QLabel" name="Header">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>412</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Machine Learning Stock Price Predictor</string>
   </property>
  </widget>
  <widget class="QLabel" name="tickerlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter Stock Symbol: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tickerbox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="daterangelabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date Range (YYYY-MM-DD)</string>
   </property>
  </widget>
  <widget class="QLabel" name="fromlabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>165</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frombox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>165</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tobox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>195</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tolabel">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>195</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSavePred">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Prediction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRunPredict">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Prediction</string>
   </property>
  </widget>
  <widget class="QLabel" name="PlotImage">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>531</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="versionlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Version 1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="desclabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>This tool uses Long Short-Term Memory neural networks to predict the path of stock price based on previous data. The true values and predicted values are plotted for visual reference.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="notelabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>470</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When Run Prediction is clicked, the plot generated will be saved as &amp;quot;plot.png&amp;quot; to the working directory of the program. It will be overwritten on the next prediction.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tiplabel1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Fitting a neural network takes time, so be patient!</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
